### DISCUSSION TITLE
"Docs are really bad", Do you agree with this?

### DISCUSSION THREAD

**Discussion by perpi** [vexxhost/atmosphere#1239] [General]
I quote the beginning of the title from here: https://www.reddit.com/r/openstack/comments/1cyrz3a/comment/l5brr4j

If you were to choose a solution for installing OpenStack and documentation was a very **important** criterion for you, would you choose Atmosphere?

**Comment by mnaser** [comment]
Hi @perpi -- thanks for bringing this up.

We have been working a lot on increasing documentation at this point:

https://vexxhost.github.io/atmosphere/

We would be happy to hear where there are missing pieces and we're adding more and more :)
